public/pages/AnomalyCharts/utils/anomalyChartUtils.tsx

```diff
@@ -190,8 +203,8 @@ export const ANOMALY_HEATMAP_COLORSCALE = [
 ];
 
 export enum AnomalyHeatmapSortType {
-  SEVERITY = 'by_severity',
-  OCCURRENCES = 'by_occurrences',
+  SEVERITY = 'severity',
+  OCCURRENCES = 'occurrence',
```

Comment:
minor but should maybe change this from OCCURRENCES to OCCURRENCE?

Reply:
Fair point, I'll change.